### PR TITLE
pom: update scm reference

### PR DESCRIPTION
**Pull request by sandrobonazzola**
update scm references pointing to GitHub instead of Gerrit

### CODE REVIEW

File: pom.xml.in [hunk 1]
```diff
@@ -57,9 +57,9 @@ limitations under the License.
     </developers>
 
     <scm>
-        <connection>scm:git:git://gerrit.ovirt.org/ovirt-dependencies.git</connection>
-        <developerConnection>scm:git:git://gerrit.ovirt.org/ovirt-dependencies.git</developerConnection>
-        <url>git://gerrit.ovirt.org/ovirt-dependencies.git</url>
+        <connection>scm:git:git@github.com:oVirt/ovirt-dependencies.git</connection>
+        <developerConnection>scm:git@github.com:oVirt/ovirt-dependencies.git</developerConnection>
+        <url>git@github.com:oVirt/ovirt-dependencies.git</url>
```
Comment (arso) [inline pom.xml.in]: I saw in other project a bit different schema for url:
```<url>https://github.com/oVirt/ovirt-dependencies</url>```

File: pom.xml.in [hunk 2]
```diff
@@ -57,9 +57,9 @@ limitations under the License.
     </developers>
 
     <scm>
-        <connection>scm:git:git://gerrit.ovirt.org/ovirt-dependencies.git</connection>
-        <developerConnection>scm:git:git://gerrit.ovirt.org/ovirt-dependencies.git</developerConnection>
-        <url>git://gerrit.ovirt.org/ovirt-dependencies.git</url>
+        <connection>scm:git:git@github.com:oVirt/ovirt-dependencies.git</connection>
+        <developerConnection>scm:git@github.com:oVirt/ovirt-dependencies.git</developerConnection>
```
Comment (arso) [inline pom.xml.in]: shouldn't it be:
```<developerConnection>scm:git:ssh://git@github.com:oVirt/ovirt-dependencies.git</developerConnection>```

File: pom.xml.in [hunk 3]
```diff
@@ -57,9 +57,9 @@ limitations under the License.
     </developers>
 
     <scm>
-        <connection>scm:git:git://gerrit.ovirt.org/ovirt-dependencies.git</connection>
-        <developerConnection>scm:git:git://gerrit.ovirt.org/ovirt-dependencies.git</developerConnection>
-        <url>git://gerrit.ovirt.org/ovirt-dependencies.git</url>
+        <connection>scm:git:git@github.com:oVirt/ovirt-dependencies.git</connection>
```
Comment (arso) [inline pom.xml.in]: shouldn't we have 
```<connection>scm:git:https://github.com/oVirt/ovirt-dependencies.git</connection>```

**Review by arso — COMMENTED** [review]
I have noticed that on other projects we have a bit different settings. Here one of the examples of the migration to  GH. Not sure what are the implications.
https://github.com/oVirt/ovirt-engine-extension-aaa-misc/commit/1b56dfc3d26dc963a79936a48d6f30803e6f2bda

**Comment by sandrobonazzola** [comment]
@arso I'm not a mvn expert, can you amend the patch?

**Comment by arso** [comment]
Sure, np. I will update it.

File: pom.xml.in (hunk 3 above)
Comment (arso) [reply pom.xml.in]: done

File: pom.xml.in (hunk 2 above)
Comment (arso) [reply pom.xml.in]: done

File: pom.xml.in (hunk 1 above)
Comment (arso) [reply pom.xml.in]: done

**Review by arso — APPROVED** [review]

**Pull request merged**